colin/checks/labels.fmf

```diff
@@ -221,3 +221,10 @@ tags: ["label"]
   labels: ["version"]
   required: True
   value_regex: Null
+
+/inherited_labels:
+  class: InheritedOptionalLabelCheck
+  message: "Optional labels are only inherited"
+  description: "Check if optional labels provided with 'labels_list' argument, are only inherited"
+  reference_url: "?????"
```

Comment:
It would be nice to have some URL with more information, do we have any?

Reply:
maybe rcm might have something, but I'd say that will be internal